src/nft/components/collection/CollectionStats.tsx

```diff
@@ -267,8 +269,8 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
         </StatsItem>
       ) : null}
       {stats.stats?.total_listings ? (
```

Comment:
Can you check with product to see if we want to show when there's `0%` listed?

Reply:
Hiding it for now until I get confirmation